src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,441 @@
+                    _visibleEventHandler = Interop.QuickPanelClient.AddEventHandler(_tzshQpClient, _visibleEventType, _onVisibleChanged, IntPtr.Zero);
+                    if (_visibleEventHandler == IntPtr.Zero)
+                    {
+                        int err = Tizen.Internals.Errors.ErrorFacts.GetLastResult();
+                        _tzsh.ErrorCodeThrow(err);
+                    }
+                }
+                _visibleChanged += value;
+            }
+            remove
+            {
+                int _res;
+                _visibleChanged -= value;
+                if (_visibleChanged == null)
+                {
+                    _res = Interop.QuickPanelClient.DelEventHandler(_tzshQpClient, _visibleEventHandler);
+                    _tzsh.ErrorCodeThrow(_res);
+                }
+            }
+        }
+
+        private int _orientationEventType;
+        private IntPtr _orientationEventHandler;
+        private Interop.QuickPanelClient.QuickPanelEventCallback _onOrientationChanged;
+        private event EventHandler<StateOrientation> _orientationChanged;
+        /// <summary>
+        /// Emits the event when the orientation of the quickpanel service window is changed.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public event EventHandler<StateOrientation> OrientationChanged
+        {
+            add
+            {
+                if (_orientationChanged == null)
+                {
+                    if (_orientationEventType == 0)
+                    {
+                        _orientationEventType = Interop.TizenShell.NewEventType(_tzsh.GetNativeHandle(), Interop.QuickPanelClient.EventStringOrientation);
+                    }
+                    _onOrientationChanged = OnOrientationChanged;
+                    _orientationEventHandler = Interop.QuickPanelClient.AddEventHandler(_tzshQpClient, _orientationEventType, _onOrientationChanged, IntPtr.Zero);
+                    if (_orientationEventHandler == IntPtr.Zero)
+                    {
+                        int err = Tizen.Internals.Errors.ErrorFacts.GetLastResult();
+                        _tzsh.ErrorCodeThrow(err);
+                    }
+                }
+                _orientationChanged += value;
+            }
+            remove
+            {
+                int _res;
+                _orientationChanged -= value;
+                if (_orientationChanged == null)
+                {
+                    _res = Interop.QuickPanelClient.DelEventHandler(_tzshQpClient, _orientationEventHandler);
+                    _tzsh.ErrorCodeThrow(_res);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Enumeration for type of quickpanel service window.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public enum QuickPanelType
```

Comment:
QuickPanelClient.QuickPanelType  => "QuickPanel" this word looks duplicated. How about QuickPanelClient.Types ?

Reply:
I changed this to Types